GOVERNANCE.md

```diff
@@ -0,0 +1,147 @@
+# Governance
+
+## Overview
+
+BlockNote is an open-source project stewarded by the founding team with the goal of growing into a contributor-led ecosystem over time. Governance is designed to balance long-term technical integrity, transparency, and sustainability while remaining lightweight at the project’s current scale.
+
+Governance is intentionally documented early to support trust with contributors, enterprises, and public-sector users.
+
+---
+
+## Core Project Roles
+
+The project uses a simple three-role structure for project governance and technical authority. These roles define who makes technical decisions and how the project is stewarded over time. Individuals may hold multiple roles, but the authority exercised depends on the role relevant to the decision at hand.
+
+The project uses this structure designed to match its current size while supporting future growth. Individuals may hold multiple roles, but the authority exercised depends on the role relevant to the decision at hand.
+
+### Technical Steering Committee (TSC)
+
+> TSC members may also serve as maintainers. When doing so, they exercise only the authority of the role relevant to the decision at hand.
+
+The Technical Steering Committee (TSC) is responsible for the long-term technical direction and health of the project.
+
+The TSC operates independently of Product Management. Product may facilitate processes and provide input, but does not exercise technical decision-making authority or override TSC decisions.
+
+**Responsibilities**:
+
+- Approving or rejecting significant technical changes (via the [RFC process](/RFC_PROCESS.md))
+- Setting and maintaining long-term architectural direction
+- Sanity-checking alignment with the project mission to prevent long-term drift
+- Acting as stewards of the project’s technical integrity and sustainability
+
+**Privileges**:
+
+- Ability to curate and prioritize problem statements
+- Ability to act as Product Owner (PO) for execution within accepted RFC scope
+- Ability to establish and facilitate regular operational processes (such as bug triage) to support maintainers and contributors
+
+Product management does not make technical decisions, does not override maintainer or TSC authority, and does not bypass the RFC or review process. Product management may facilitate processes such as bug triage, but technical classification and severity decisions remain the responsibility of maintainers. Individuals may hold multiple roles; decision authority depends on the role being exercised.
```

Comment:
You're introducing the term "Product management" under the section about TSC.

I think that if these are purely clarifications about PM vs. TSC, I think they should be more succint. If they're here to describe the role of PM, it should move to the section below.